docs/tools/generate.fsx

```diff
@@ -22,6 +22,7 @@ let info =
 // For typical project, no changes are needed below
 // --------------------------------------------------------------------------------------
 
+#I "../../packages/FAKE/tools/"
```

Comment:
Sorry, I don't understand. Please open a separate PR describing what the problem is you're trying to solve, and how this solves it.